src/pages/register.tsx

```diff
@@ -0,0 +1,216 @@
+      <Logo width={119} height={119} />
+
+      <Box style={{ width: "50vw" }}>
+        <CommonTextField
+          header="Username"
+          placeholder="Display Name"
+          value={displayName}
+          handleValueChange={handleDisplayNameChange}
+          char_limit={100}
+          isErr={isSubmit && displayNameErr.err}
+          errMsg={displayNameErr.msg}
+        />
+      </Box>
+
+      <Box style={{ width: "50vw", marginTop: 0 }}>
+        <CommonTextField
+          header="Email"
+          placeholder="Email"
+          value={email}
+          handleValueChange={handleEmailChange}
+          isErr={isSubmit && emailErr.err}
+          errMsg={emailErr.msg}
+        />
+      </Box>
+
+      <Box style={{ width: "50vw" }}>
+        <PasswordTextFeild
+          header="Password"
+          placeholder="Password"
+          value={password}
+          handleValueChange={handlePasswordChange}
+          isErr={isSubmit && passwordErr.err}
+          errMsg={passwordErr.msg}
+        />
+      </Box>
+
+      <Box style={{ width: "50vw" }}>
+        <PasswordTextFeild
+          header="Confirm Password"
+          placeholder="Confirm Password"
+          value={confirmPassword}
+          handleValueChange={handleConfirmPasswordChange}
+          isErr={isSubmit && confirmPasswordErr.err}
+          errMsg={confirmPasswordErr.msg}
+        />
+      </Box>
+
+      <Grid item width={"50vw"}>
+        <Grid container spacing={3} justifyContent="left">
+          <Grid item xs={6}>
+            <CommonDropdown
+              header="Gender"
+              placeHolder="Gender"
+              value={gender}
+              handleValueChange={handleSelectChange}
+              items={Object.values(Gender)}
+            />
+            <Box sx={helperText}>
+              {!isValidGender && (
+                <FormHelperText error>{"ช่องนี้ไม่สามารถเว้นว่างได้"}</FormHelperText>
+              )}
+            </Box>
+          </Grid>
+
+          <Grid item xs={6}>
+            <Typography>Birth Date</Typography>
+            <LocalizationProvider dateAdapter={AdapterDayjs}>
+              <MobileDatePicker
+                inputFormat="DD/MM/YYYY"
+                mask="__/__/____"
+                value={birthDate}
+                disableFuture={true}
+                onChange={(newBirthDate) => {
+                  setBirthDate(newBirthDate);
+                }}
+                renderInput={(params) => <TextField {...params} fullWidth/>}
+                InputProps={{
+                  startAdornment: (
+                    <InputAdornment position="start">
+                      <CalendarTodayIcon />
+                    </InputAdornment>
+                  ),
+                }}
+              />
+            </LocalizationProvider>
+            <Box sx={helperText}>
+              {!isValidBirthDate && (
+                <FormHelperText error>{"ช่องนี้ไม่สามารถเว้นว่างได้"}</FormHelperText>
```

Comment:
I think you still missing error "Birth Date ไม่ถูกต้อง" which occurs when user type in the invalid date instead of choosing from date picker
<img width="228" alt="image" src="https://user-images.githubusercontent.com/62459518/218260407-a2016c2d-1fbd-4d83-b029-e4feddddb72d.png">